config.json

```diff
@@ -259,7 +259,11 @@
     {
       "difficulty": 1,
       "slug": "luhn",
-      "topics": []
+      "topics": [
+	"Algorithms",
+	"Integers",
+	"Strings"
```

Comment:
I can agree with that. Do you think anything in the track besides hello world would be a 1? Or should we look at anything more complex than that as at least a 2.

(I requested help just to see what would happen. I hadn't noticed that button before. All it does is put that flag on the solution submission?)